atlasdb-dbkvs/src/main/java/com/palantir/atlasdb/keyvalue/dbkvs/OracleDdlConfig.java

```diff
@@ -99,4 +99,11 @@ protected final void checkOracleConfig() {
                 "Oracle 'overflowTablePrefix' cannot be more than 6 characters long.");
 
     }
+
+    @Value.Default
+    @Override
+    public int fetchBatchSize() {
+        // Oracle Arrays can be much larger than Postgres select-where-in way of doing get
+        return 1024;
```

Comment:
I didn't do any thorough benchmarking of this yet, except for thorough benchmarking of the "before this change" part.
The real trouble is that I know I can't do the batch tuning with my current hydration test setup, since the latency of my laptop to my IL VM running Oracle docker image is totally out of whack with the reality of our actual deployments.
... and local Oracle is a PITA.